client/src/locales/fr.ts

```diff
@@ -0,0 +1,318 @@
+			text3: "Actuellement, vous pouvez regarder des vidéos depuis : Youtube, Vimeo, Dailymotion, liens directs vers des vidéos .mp4, ...",
+			link: "d'autres sources arrivent bientôt",
+		},
+		features: {
+			"title": "Fonctionnalités Principales",
+			"syncronized-playback": {
+				title: "Vidéo synchronisée",
+				text: "Appuies sur le bouton play et la vidéo se lance pour tout le monde dans la salle. Aussi simple que ça.",
+			},
+			"permanent-rooms": {
+				title: "Salles Permanentes",
+				text: "Toi et t'es amis êtes des utilisateurs réguliers ? Tu peux créer une salle permanente qui te permettra de conserver la même URL pour ta salle",
+			},
+			"dark-theme": {
+				title: "Theme Sombre",
+				text: "Tu regardes des vidéos tard le soir avec t'es copains ? Evitez de vous abimer les yeux avec le Theme Sombre",
+			},
+			"room-permissions": {
+				title: "Permissions de salle",
+				text: "Fini les trolls ou tout autre problème dû à des gens mal intentionnés. Gère toutes les permissions de ta salle pour qu'ils ne puissent plus vous embêter.",
+			},
+			"voting-system": {
+				title: "Système de vote",
+				text: "Vous avez du mal à choisir quelle sera la prochaine vidéo à être regardée ? Passe du mode liste de lecture au système de vote pour que la vidéo la plus votée soit visionnée avant les autres.",
+			},
+			"playlist-copying": {
+				title: "Ajoute des Playlist",
+				text: "Ajoute une playlist ou carrément une chaîne entière dans la liste de lecture afin que tu ne sois pas obligé d'ajouter une par une les vidéos, c'est le meilleur moyen de découvrir une chaîne avec t'es amis.",
+			},
+		},
+		support: {
+			title: "Soutenir le développement",
+			description1:
+				"OpenTogetherTube n'existerait pas sans l'aide et les contributions des utilisateurs comme toi.",
+			description2:
+				"Vous pouvez nous aider en nous partageant vos idées ou même votre codage, vous pouvez montrer votre soutient en devenant un sponsor. Tous les dons sont utilisés pour payer les frais d'hebergement, pour le développement d'OpenTogetherTube et pour garder OpenTogetherTube sans publicité",
+			how: "Comment puis-je aider ?",
+			sponsor: "Devenir un sponsor",
+			contribute: "Contribuer",
+		},
+	},
+	"footer": {
+		"disclaimer":
+			"Information importante: OpenTogetherTube n'est pas associé à TogetherTube ou Watch2Gether.",
+		"made-in": "Made in America",
+		"thanks-to": "Remerciments spécial à",
+		"privacy-policy": "Politique de Confidentialité",
+		"attribution": "Attribution",
+	},
+	"not-found": {
+		title: "Page non trouvée",
+		home: "@:nav.home",
+		browse: "@:landing.hero.btns.browse",
+	},
+	"quick-room": {
+		text: "Création d'une salle temporaire pour vous...",
+	},
+	"attribution": {
+		"sponsorblock-text": "Utilise les données SponsorBlock de",
+	},
+	"nav": {
+		"home": "Accueil",
+		"browse": "Parcourir",
+		"faq": "FAQ",
+		"bug": "Rapporter un BUG",
```

Comment:
This doesn't seem right, should "BUG" be in all caps here?

Also, google translate has a contributor reviewed translation for "Report a bug" -> "Signaler un bug". What's the difference between "Rapporter un bug" and "Signaler un bug"?

Reply:
Yes indeed for the bug, it is better to put it in lowercase. No difference between "Rapporter un bug" and "Signaler un bug".